src/cli/cli_mdns.cpp

```diff
@@ -459,6 +482,117 @@ template <> otError Mdns::Process<Cmd("unregister")>(Arg aArgs[])
     return error;
 }
 
+template <> otError Mdns::Process<Cmd("hosts")>(Arg aArgs[])
```

Comment:
This is just a preference: but what if we group all these commands into one: ```mdns list hosts|services|keys```?

Reply:
It's not a hotpath ... so just increasing the time with 10 cycles it's negligible.